### ISSUE TITLE
Big background image causes crash on iPad

### ISSUE THREAD

**Issue by jperedadnr** [HanSolo/SpaceFX#6]
SpaceFX works fine on iPhone, however, when deploying to an iPad (Mini 4), it crashes due to a very big background image:

```
new Image(getClass().getResourceAsStream("backgroundL1.jpg"), 700 * BKG_SCALING_FACTOR, 3379 * BKG_SCALING_FACTOR
```
iPad visual bounds: 768x1004, image size: 781x3769. It is not that big, but it seems there is no enough memory.

**Comment by HanSolo** [comment]
Yep I never planned to bring it to the iPad because future apps on iOS for the iPad need to support the new split mode which won‘t be supported by Gluon I guess?
